cmd/src/batch_remote.go

```diff
@@ -100,6 +100,16 @@ Examples:
 		}
 		ui.SendingBatchSpecSuccess()
 
+		ui.UploadingWorkspaceFiles()
+		dir, err := getBatchSpecDirectory(file)
+		if err != nil {
+			return errors.Wrap(err, "batch spec path")
+		}
+		if err = svc.UploadBatchSpecWorkspaceFile(dir, batchSpecID, spec.Steps); err != nil {
+			return err
+		}
+		ui.UploadingWorkspaceFilesSuccess()
```

Comment:
would be nice to only log the messages here when there are actually any file matches, otherwise it might be confusing "Oh no what is it uploading??"